Mathlib/Algebra/Exact.lean

```diff
@@ -10,13 +10,13 @@ import Mathlib.Algebra.Module.Submodule.Range
 /-! # Exactness of a pair
 
 * For two maps `f : M → N` and `g : N → P`, with `Zero P`,
-`Function.AddExact f g` says that `Set.range f = Set.preimage {0}
+`Function.AddExact f g` says that `Set.range f = Set.preimage {0}`
```

Comment:
Shouldn't this be `Set.preimage g {0}`?

Reply:
Yes, but also this should be `Function.Exact`, and `Function.AddExact` doesn't exist (cc @AntoineChambert-Loir)